api/src/main/java/org/commonjava/indy/conf/DefaultIndyConfiguration.java

```diff
@@ -407,6 +411,17 @@ public void setStoreManagerStandalone( Boolean storeManagerStandalone )
         this.storeManagerStandalone = storeManagerStandalone;
     }
 
+    public int getKeyspacesReplica()
+    {
+        return this.keyspacesReplica == null ? DEFAULT_CASSANDRA_KEYSPACE_REPLICA : keyspacesReplica ;
+    }
+
+    @ConfigName( "cassandra.keyspaces.replica" )
```

Comment:
This is a minor point, but can we make this `cassandra.keyspace.replicas`, and change the getter/setter names to match? I found this a little confusing (not much, but it's the kind of confusion that I think could stack up over time for new users/developers).

Reply:
Sure, good point. thanks.